### PR TITLE
July 2021 updates

### PR DESCRIPTION
**Pull request by npaterno**
One data set per commit. Let me know if this format works well for larger updates like this. 

Thanks!
~N

### CODE REVIEW

File: R/data-baby_names.R [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#' Baby Names Data.
```
Comment (mine-cetinkaya-rundel) [inline R/data-baby_names.R]: How does this relate to https://cran.r-project.org/web/packages/babynames/index.html? Is it worth doing a PR there to update instead?

File: R/data-blizzard_salary.R [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+#' @examples 
+#' library(ggplot2)
+#' library(dplyr)
+#'
+#'plot_data <- blizzard_salary %>% 
+#'mutate(annual_salary = case_when(
+#'salary_type == "week" ~ current_salary*52,
+#'salary_type == "hour" ~ current_salary*40*52,
+#'TRUE ~ current_salary
+#'))
```
Comment (mine-cetinkaya-rundel) [inline R/data-blizzard_salary.R]:
```suggestion
#'plot_data <- blizzard_salary %>% 
#'  mutate(annual_salary = case_when(
#'    salary_type == "week" ~ current_salary*52,
#'    salary_type == "hour" ~ current_salary*40*52,
#'.   TRUE ~ current_salary
#'  )
#')
```

File: R/data-blizzard_salary.R [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+#'
+#'ggplot(plot_data, aes(annual_salary))+
+#'geom_histogram(binwidth = 25000, color = "white")+
+#'labs(title = "Current Salary of Blizzard Employees",
+#'     x = "Salary",
+#'          y = "Number of Employees")
```
Comment (mine-cetinkaya-rundel) [inline R/data-blizzard_salary.R]:
```suggestion
#'labs(
#'  title = "Current Salary of Blizzard Employees",
#'  x = "Salary",
#'  y = "Number of Employees"
#')
```

File: R/data-mlb_teams.R [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+#' # List the World Series winning teams for each year
+#' mlb_teams %>% 
+#'   filter(world_series_winner=="Y")%>%
+#'   select(year,team_name,ball_park)
+#'
+#' # List the teams with their average number of wins and losses
+#'  mlb_teams %>%
+#'    group_by(team_name)%>%
+#'    summarize(mean_wins=mean(wins),mean_losses=mean(losses))%>%
+#'    arrange((team_name))
```
Comment (mine-cetinkaya-rundel) [inline R/data-mlb_teams.R]:
```suggestion
#' mlb_teams %>%
#'   group_by(team_name)%>%
#'   summarize(mean_wins=mean(wins),mean_losses=mean(losses))%>%
#'   arrange((team_name))
```

File: R/data-reddit_finance.R [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+#' 
+#' @examples
+#' 
+#' library(ggplot2)
+#' # Histogram of Expected Retirement Age.
+#' ggplot(reddit_finance, aes(retire_age))+
+#' geom_bar(na.rm = TRUE)+
+#' labs(title = "At what age do you expect to retire?",
+#' x = "Age Bracket",
+#' y = "Number of Respondents")
```
Comment (mine-cetinkaya-rundel) [inline R/data-reddit_finance.R]:
```suggestion
#' geom_bar(na.rm = TRUE)+
#'   labs(
#'     title = "At what age do you expect to retire?",
#'     x = "Age Bracket",
#'     y = "Number of Respondents"
#' )
```

**Review by mine-cetinkaya-rundel — CHANGES_REQUESTED** [review]
I've made some suggestions for code formatting, though it might be more efficient to update them throughout and then I can take another look. A few things to keep in mind:

- There should be a space after `#'` and the code for the exercise
- There should be a line break after `+` in ggplots
- The styler package can be helpful for automatic styling

File: R/data-baby_names.R (hunk 1 above)
Comment (npaterno) [reply R/data-baby_names.R]: I'll look into this!

**Comment by npaterno** [comment]
I'll make the requested changes and go through all the other code for this pr and for the pr in usdata.

File: R/data-baby_names.R (hunk 1 above)
Comment (npaterno) [reply R/data-baby_names.R]: Looks like it should definitely be there to update instead of here.

Comment (mine-cetinkaya-rundel) [reply R/data-baby_names.R]: Might be worth opening an issue there to see if Hadley is interested in an update.

Comment (npaterno) [reply R/data-baby_names.R]: Will do!

**Comment by npaterno** [comment]
All documentation files should now be formatted correctly. I also added the @name, @docType and @keywords fields since they were included more often than not in documentation for other datasets.

**Review by mine-cetinkaya-rundel — APPROVED** [review]
Looks good!

**Pull request merged**